### PR TITLE
i18n: translate more 7.x content

### PR DESCRIPTION
**Pull request by Akurosia**
regarding some entries in Valigarmanda: 
`47.2 "Slithering Strike/Strangling Coil" Ability { id: ["8D3D", "8D3F"], source: "Valigarmanda" }` here we habe 2 abillities fully written out but 
`1603.9 "Calamitous Cry/Echo" Ability { id: "8D60", source: "Valigarmanda" }` here its split.

In the past there was a discussion how to deal with it and usually we but both names written out fully and handle the shorthand version via the translations like it was made in `1072.3 "Strangling Coil/Slithering Strike/Susurrant Breath" Ability { id: ["8D3B", "8D3D", "8D3F"], source: "Valigarmanda" }`

Please let me know if i should cover this in this PR or in a seperate one

### CODE REVIEW

**Comment by Akurosia** [comment]
Additionally what i just found:
```
1603.9 "Calamitous Cry/Echo" Ability { id: "8D60", source: "Valigarmanda" }
1609.9 "Calamitous Cry/Echo" Ability { id: "8D61", source: "Valigarmanda" }
1615.9 "Calamitous Cry/Echo" Ability { id: "8D61", source: "Valigarmanda" }
1621.9 "Calamitous Cry/Echo" Ability { id: "8D61", source: "Valigarmanda" }
1627.9 "Calamitous Cry/Echo" Ability { id: "8D61", source: "Valigarmanda" }
1633.9 "Calamitous Cry/Echo" Ability { id: "8D61", source: "Valigarmanda" }
1639.9 "Calamitous Cry/Echo" Ability { id: "8D61", source: "Valigarmanda" } forcejump 1609.9
```

This block in Valigarmanda Has 2 Ability Names but only 1 ID. Also the ID for the Echo is not in the list.
Shouldn't it either: 1. include both IDs on all lines OR 2. just include 2 lines 1 for each id with a loop and only the used name for the ID ?
8D60/8D61 = Calamitous Cry
8D63 = Calamitous Echo

**Comment by xiashtra** [comment]
> regarding some entries in Valigarmanda: `47.2 "Slithering Strike/Strangling Coil" Ability { id: ["8D3D", "8D3F"], source: "Valigarmanda" }` here we habe 2 abillities fully written out but `1603.9 "Calamitous Cry/Echo" Ability { id: "8D60", source: "Valigarmanda" }` here its split.
> 
> In the past there was a discussion how to deal with it and usually we but both names written out fully and handle the shorthand version via the translations like it was made in `1072.3 "Strangling Coil/Slithering Strike/Susurrant Breath" Ability { id: ["8D3B", "8D3D", "8D3F"], source: "Valigarmanda" }`
> 
> Please let me know if i should cover this in this PR or in a seperate one

Yes. we should stick with writing out the full name in timelines and then using `replaceText` for shortening.  It was done that way to make things easier on translators where the EN shortening didn't make sense or was harder to translate in other languages.

**Comment by xiashtra** [comment]
> Additionally what i just found:
> 
> ```
> 1603.9 "Calamitous Cry/Echo" Ability { id: "8D60", source: "Valigarmanda" }
> 1609.9 "Calamitous Cry/Echo" Ability { id: "8D61", source: "Valigarmanda" }
> 1615.9 "Calamitous Cry/Echo" Ability { id: "8D61", source: "Valigarmanda" }
> 1621.9 "Calamitous Cry/Echo" Ability { id: "8D61", source: "Valigarmanda" }
> 1627.9 "Calamitous Cry/Echo" Ability { id: "8D61", source: "Valigarmanda" }
> 1633.9 "Calamitous Cry/Echo" Ability { id: "8D61", source: "Valigarmanda" }
> 1639.9 "Calamitous Cry/Echo" Ability { id: "8D61", source: "Valigarmanda" } forcejump 1609.9
> ```
> 
> This block in Valigarmanda Has 2 Ability Names but only 1 ID. Also the ID for the Echo is not in the list. Shouldn't it either: 1. include both IDs on all lines OR 2. just include 2 lines 1 for each id with a loop and only the used name for the ID ? 8D60/8D61 = Calamitous Cry 8D63 = Calamitous Echo

@wexxlee should probably chime in on if there is a missing ID here.  It looks like these changes might be worth splitting off into a clean-up PR.

**Comment by xiashtra** [comment]
The Hunt translations overlap with #386, can you take a look at that PR and see if it covers everything you have here?

**Comment by Akurosia** [comment]
> The Hunt translations overlap with #386, can you take a look at that PR and see if it covers everything you have here?

I made a comment there, maybe you can give your 2 cents there as well just so we are all on a level how to deal with non implemented stuff for translations

File: ui/raidboss/data/07-dt/trial/valigarmanda.txt [hunk 1]
```diff
@@ -71,13 +71,13 @@ hideall "--sync--"
...
-1627.9 "Calamitous Cry/Echo" Ability { id: "8D61", source: "Valigarmanda" }
-1633.9 "Calamitous Cry/Echo" Ability { id: "8D61", source: "Valigarmanda" }
-1639.9 "Calamitous Cry/Echo" Ability { id: "8D61", source: "Valigarmanda" } forcejump 1609.9
+1603.9 "Calamitous Cry/Calamitous Echo" Ability { id: "8D60", source: "Valigarmanda" }
+1609.9 "Calamitous Cry/Calamitous Echo" Ability { id: "8D61", source: "Valigarmanda" }
+1615.9 "Calamitous Cry/Calamitous Echo" Ability { id: "8D61", source: "Valigarmanda" }
+1621.9 "Calamitous Cry/Calamitous Echo" Ability { id: "8D61", source: "Valigarmanda" }
+1627.9 "Calamitous Cry/Calamitous Echo" Ability { id: "8D61", source: "Valigarmanda" }
+1633.9 "Calamitous Cry/Calamitous Echo" Ability { id: "8D61", source: "Valigarmanda" }
+1639.9 "Calamitous Cry/Calamitous Echo" Ability { id: "8D61", source: "Valigarmanda" } forcejump 1609.9
```
Comment (wexxlee) [inline ui/raidboss/data/07-dt/trial/valigarmanda.txt:80]:
```suggestion
1603.9 "Calamitous Cry + Calamitous Echo" Ability { id: "8D60", source: "Valigarmanda" }
1609.9 "Calamitous Cry + Calamitous Echo" Ability { id: "8D61", source: "Valigarmanda" }
1615.9 "Calamitous Cry + Calamitous Echo" Ability { id: "8D61", source: "Valigarmanda" }
1621.9 "Calamitous Cry + Calamitous Echo" Ability { id: "8D61", source: "Valigarmanda" }
1627.9 "Calamitous Cry + Calamitous Echo" Ability { id: "8D61", source: "Valigarmanda" }
1633.9 "Calamitous Cry + Calamitous Echo" Ability { id: "8D61", source: "Valigarmanda" }
1639.9 "Calamitous Cry + Calamitous Echo" Ability { id: "8D61", source: "Valigarmanda" } forcejump 1609.9
```

File: ui/raidboss/data/07-dt/trial/valigarmanda.ts [hunk 2]
```diff
@@ -403,9 +407,104 @@ const triggerSet: TriggerSet<Data> = {
     {
       'locale': 'en',
       'replaceText': {
+        'Calamitous Cry/Calamitous Echo': 'Calamitous Cry/Echo',
         'Strangling Coil/Slithering Strike/Susurrant Breath': 'Middle/Away/Front Corners',
```
Comment (wexxlee) [inline ui/raidboss/data/07-dt/trial/valigarmanda.ts:411]:
```suggestion
        'Calamitous Cry + Calamitous Echo': 'Calamitous Cry + Echo',
        'Slithering Strike/Strangling Coil': 'Middle/Away',
        'Strangling Coil/Slithering Strike/Susurrant Breath': 'Middle/Away/Front Corners',
```

**Review by wexxlee — CHANGES_REQUESTED** [review]
Per earlier comment

**Review by wexxlee — APPROVED** [review]

**Comment by wexxlee** [comment]
@Akurosia looks like some branch conflicts due to the merge activity this morning - should be good to merge after those are resolved.  Thx!

**Pull request merged**